demo/addons/gdhtml/gdhtml.gdextension

```diff
@@ -5,18 +5,36 @@ compatibility_minimum = "4.3"
 
 [dependencies]
 
-macos.debug.arm64 = {
-    "bin/macos/arm64/libUltralight.dylib" : "Contents/Frameworks",
-    "bin/macos/arm64/libUltralightCore.dylib" : "Contents/Frameworks",
-    "bin/macos/arm64/libWebCore.dylib" : "Contents/Frameworks",
-    "bin/macos/arm64/libAppCore.dylib" : "Contents/Frameworks",
+macos.debug = {
+    "bin/macos/arm64/libUltralight.dylib" : "Contents/Frameworks/arm64",
+    "bin/macos/arm64/libUltralightCore.dylib" : "Contents/Frameworks/arm64",
+    "bin/macos/arm64/libWebCore.dylib" : "Contents/Frameworks/arm64",
+    "bin/macos/arm64/libAppCore.dylib" : "Contents/Frameworks/arm64",
+
+    "bin/macos/arm64/gdhtml.macos.template_debug.arm64.dylib": "Contents/Frameworks/addons/gdhtml/bin/macos/arm64",
```

Comment:
This line (and the equivalent one for x86_64) is needed because for some reason the exported game tries to look for the gdextension library in the `addons/gdhtml` folder inside of the Mac app. I don't think this is really the right thing to do, nor do I know enough about how Godot works on Mac to figure out any alternatives, but it works for me right now.